gbus/messages.go

```diff
@@ -96,6 +107,7 @@ func (bm *BusMessage) GetTraceLog() (fields []log.Field) {
 		log.String("SagaCorrelationID", bm.SagaCorrelationID),
 		log.String("Semantics", string(bm.Semantics)),
 		log.String("RPCID", bm.RPCID),
+		log.String("ContentType", bm.ContentType),
```

Comment:
The `ContentType` should come from the serializer and not from the message itself since it is not the one that handles the serialization

Reply:
we have gob serializer, that if I understood correctly can support several types of messages, if this is the case we need to be able to change the contentType, no?

Reply:
@hadascoh, as Vlad mentioned the ContentType, should come from the serializer....if you plan to consume and publish messages using a different wire format then you need to set the correct serialzier for that

Reply:
So we need to create a new serializer that fits those issues?